code/__HELPERS/turfs.dm

```diff
@@ -16,10 +16,15 @@
 //This proc is used in only two places, ive changed it to make more sense
 //The old behaviour returned zero if there were any simulated atoms at all, even pipes and wires
 //Now it just finds if the tile is blocked by anything solid.
+//Edit by SCPR - 2022
+//Made it check for cables and pipes , because having pipes / cables randomly unwired because of it can be very fustrating
+//For technomancers.
```

Comment:
```suggestion
//Edit by SCPR - 2022
//Made it check for cables and pipes, because having pipes / cables randomly unwired because of it can be very frustrating for technomancers.
```